src/UsedSymbolExtractor.php

```diff
@@ -231,6 +229,9 @@ public function parseUseStatement(): array
                         $alias = $classSplit[count($classSplit) - 1];
                         break;
 
+                    case T_FUNCTION:
+                        break;
```

Comment:
This needs to set sth like `$useKind = self::KIND_FUNCTION` and then instead of `$statements[$alias]` we use `$statements[$useKind][$alias]`

Reply:
Most probably yes. This is a draft.